db/migrate/20241030205550_add_columns_to_users.rb

```diff
@@ -0,0 +1,11 @@
+class AddColumnsToUsers < ActiveRecord::Migration[6.1]
+  def change
+    add_column :users, :alt_first_name, :string
+    add_column :users, :alt_last_name, :string
+    add_column :users, :credentials, :text
+    add_column :users, :work, :text
+    add_column :users, :project, :text
+    add_column :users, :alt_job_title, :string
+    add_column :users, :fellowship, :string
+  end
```

Comment:
```suggestion
    add_column :users, :fellowship, :string
  end
```
Reviewed the designs - this fellowships field should be removed. It maps to the same content as `alt_job_title`.